packages/components/src/select/tests/chromatic/Select.stories.tsx

```diff
@@ -1,95 +1,54 @@
 import { Inline, Stack } from "@components/layout";
 import { Select, useSelect } from "@components/select";
-import { paramsBuilder, storiesOfBuilder } from "@stories/utils";
 
 import { Button } from "@components/button";
 import { Div } from "@components/html";
 import { Item } from "@components/collection";
 import { Listbox } from "@components/listbox";
 import { Overlay } from "@components/overlay";
-import { createTestSuite } from "./createTestSuite";
 
-function stories(segment) {
-    return storiesOfBuilder(module, "Chromatic/Select")
-        .segment(segment)
-        .parameters(paramsBuilder()
-            .chromaticDelay(100)
-            .chromaticPauseAnimationAtEnd()
-            .build())
-        .build();
-}
+import { Meta, StoryObj } from "@storybook/react";
 
-createTestSuite(<Select variant="outline" />, stories("/outline"))
-    .add("without placeholder", () =>
-        <Select aria-label="Planets">
-            <Item key="earth">Earth</Item>
-            <Item key="mars">Mars</Item>
-            <Item key="saturn">Saturn</Item>
-        </Select>
-    );
+const meta = {
+    title: "Chromatic/Select",
+    component: Select,
+    parameters: {
+        chromatic: {
+            delay: 100,
+            pauseAnimationAtEnd: true
```

Comment:
I would validate that this still works (pauseAnimationAtEnd: true) by comparing the old and the new snapshot